net.sf.eclipsecs.core/src/net/sf/eclipsecs/core/config/meta/MetadataFactory.java

```diff
@@ -330,9 +570,15 @@ public static synchronized void refresh() {
    *           error loading the meta data file
    */
   private static void doInitialization() throws CheckstylePluginException {
+    createPropertyTypeMapping();
+    createPackageToGroupNameMapping();
 
     ClassLoader classLoader = CheckstylePlugin.getDefault().getAddonExtensionClassLoader();
     Collection<String> potentialMetadataFiles = getAllPotentialMetadataFiles(classLoader);
+
+    getThirdPartyModuleExtensionMetadata();
```

Comment:
please rename this method
it is names as `get....` but no result from method. Did you mean `load` ?

Reply:
Done